packages/fhir-care-team/src/components/CreateEditCareTeam/index.tsx

```diff
@@ -55,6 +55,8 @@ const CreateEditCareTeam: React.FC<CreateEditCareTeamProps> = (props: CreateEdit
       onError: () => sendErrorNotification(t('An error occurred')),
       select: (res) => res,
       enabled: !!careTeamId,
+      cacheTime: 0,
+      staleTime: 0,
```

Comment:
why?

Reply:
This makes sure, the careTeam data fetched is always the latest. So imagine a scenario where this request takes a while to resolve and the user, edits it the first time , and then immediately tries to edit again,

Reply:
Can this be solved by a loader or an `is updating` banner?

Reply:
The loader would be a consequence of this change, using a banner is an alternative, though abit tricky to do , since we need merge any user changes done whilst the banner is displayed, with the newly fetched details of the careteam should it be different.